src/Model/GameSystem.rb

```diff
@@ -12,8 +12,12 @@ def start_game(colour1, colour2)
     @turn_operator = TurnOperator.new(@player1, @player2)
   end
 
-  def end_game
-    FanoronaLogger.log_error('Not Implemented')
+  def end_game(winner)
+    FanoronaLogger.log_info
+    colour = winner.check_colour
+
+    puts "Player #{colour} is the Winner!!"
```

Comment:
I don't wanna be picky but `Player :B is the Winner!!` does not make very much sense 😭

Reply:
changed to player num